.github/workflows/pr-reviewed.yml

```diff
@@ -24,6 +24,9 @@ on:
 jobs:
   remove-triage:
```

Comment:
Given the changes introduced by this PR, we may need to consider renaming this

Reply:
That's a good point. I realized the workflow file names are a bit incorrect too, so I rearranged some things. PTAL